app/konnect/dev-portal/index.md

```diff
@@ -3,15 +3,15 @@ title: Dev Portal Overview
 content_type: explanation
 ---
 
-The {{site.konnect_short_name}} Dev Portal is a customizable website for developers to locate, access, and consume API services. The Dev Portal enables developers to browse and search API documentation, test API endpoints, and manage their own credentials. {{site.konnect_short_name}} offers flexible deployment options to support both internal and external APIs that can be managed from the same {{site.konnect_short_name}} instance.
+The {{site.konnect_short_name}} Dev Portal is a customizable website for developers to locate, access, and consume API services. The Dev Portal enables developers to browse and search API documentation, test API endpoints, and manage their own credentials. {{site.konnect_short_name}} offers flexible deployment options to support both internal and external APIs that can be managed from {{site.konnect_short_name}}.
 
 ## Publish service documentation
 
-In {{site.konnect_short_name}}, API products can be published to the Dev Portal with one click. Published services become immediately available to users who have access to the Dev Portal. When a service is published, the API spec and any service document become discoverable. We use the term discoverable here because the Dev Portal can create a unified API experience where a developer can navigate through the different APIs that are available, read documentation, test endpoints within the Dev Portal, and register to create applications using your specific APIs.
+In {{site.konnect_short_name}}, API products can be published to the Dev Portal with one click. Published products become immediately available to users who have access to the Dev Portal. When an API product is published, the API spec and any service document become discoverable. We use the term discoverable here because the Dev Portal can create a unified API experience where a developer can navigate through the different APIs that are available, read documentation, test endpoints within the Dev Portal, and register to create applications for specific APIs. 
```

Comment:
```suggestion
In {{site.konnect_short_name}}, API products can be published to the Dev Portal with one click. Published API products become immediately available to users who have access to the Dev Portal. When an API product is published, the API spec and any service documentation becomes discoverable. We use the term discoverable here because the Dev Portal can create a unified API experience where a developer can navigate through the different APIs that are available, read documentation, test endpoints within the Dev Portal, and register to create applications for specific APIs. 
```